load_uploaded_files.json

```
@@ -21,15 +21,15 @@
         },
         {
             "name": "format",
-            "description": "The file format to read from. It must be one of the values that the server reports as supported input file formats, which usually correspond to the short GDAL/OGR codes. This parameter is *case insensitive*.",
+            "description": "The file format to read from. It must be one of the values that the server reports as supported input file formats, which usually correspond to the short GDAL/OGR codes. Otherwise a `ProcessParameterInvalid` exception will be thrown. If the format is not suitable for loading the data, a `FormatUnsuitable` exception will be thrown. This parameter is *case insensitive*.",
             "schema": {
                 "type": "string",
                 "subtype": "input-format"
             }
         },
         {
             "name": "options",
-            "description": "The file format options to be used to read the files. Must correspond to the options that the server reports as supported options for the chosen `format`. The option names and valid values usually correspond to the GDAL/OGR format options.",
+            "description": "The file format parameters to be used to read the files. Must correspond to the parameters that the server reports as supported parameters for the chosen `format`. The option names and valid values usually correspond to the GDAL/OGR format options. If invalid parameters are given, a `ProcessParameterInvalid` exception will be thrown.",
```

```suggestion
            "description": "The file format parameters to be used to read the files. Must correspond to the parameters that the server reports as supported parameters for the chosen `format`. The parameter names and valid values usually correspond to the GDAL/OGR format options.",
```

I wouldn't speak about `ProcessParameterInvalid`, otherwise we could add that text to all parameters.